### PR TITLE
small changes, "backend" is kept

### PR DESCRIPTION
**Pull request by franzholz**
The extension key can have an underline character "_ " inside of its name.

### CODE REVIEW

**Comment by franzholz** [comment]
The terms "frontend" and "backend" do not exist in any English dictionary. They are only German Denglish words. It is written wrongly on many places.

Look here:

https://www.thefreedictionary.com/Backend
https://en.bab.la/dictionary/english-german/backend
https://www.oxfordlearnersdictionaries.com/definition/english/back-end_2?q=backend
https://www.dictionary.com/misspelling?term=backend&s=t
https://de.langenscheidt.com/englisch-deutsch/search?term=backend&q_cat=%2Fenglisch-deutsch%2F
https://www.dict.cc/?s=backend

**Comment by nicodh** [comment]
Yes I know but the question is if we stick to the (wrong) TYPO3 wording convention since our context is the TYPO3 eco system...

**Comment by simonschaufi** [comment]
I always thought that it is one word

**Comment by liayn** [comment]
There has been a decision that these artificially crafted words Frontend and Backend should be used like this. Please contact the documentation team on Slack to cross-check with their knowledge and guidelines.

**Comment by gilbertsoft** [comment]
Also see https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/GeneralConventions/Glossary.html

**Comment by DavidBruchmann** [comment]
> There has been a decision that these artificially crafted words Frontend and Backend should be used like this.

yes the page linked by @gilbertsoft explains they shall be used not capitalized.

I'd say, back to the end - front or back ;-)

File: Documentation/Developer/Index.rst [hunk 1]
```diff
@@ -80,29 +80,29 @@ Adding plugins
 Every extension that should generate output in the frontend needs a plugin. It will be shown in the "plugin type" field
 of the content element "Plugin". You can add it to your extension in the left panel:
 
-+--------+--------------------------------------------------------+
++--------+---------------------------------------------------------+
 |**Name**|A short, speaking name that is displayed in the backend |
-+--------+--------------------------------------------------------+
-|**Key** |A unique key to identify the plugin in your extension   |
-+--------+--------------------------------------------------------+
++--------+---------------------------------------------------------+
```
Comment (liayn) [inline Documentation/Developer/Index.rst:85]: This is a broken table layout

File: Documentation/Developer/Index.rst [hunk 2]
```diff
@@ -80,29 +80,29 @@ Adding plugins
...
+|**Key** |A unique key to identify the plugin in your extension    |
++--------+---------------------------------------------------------+
 
 
 Adding backend modules
-``````````````````````
+```````````````````````
```
Comment (liayn) [inline Documentation/Developer/Index.rst:91]: one too much

File: Documentation/Developer/Index.rst [hunk 3]
```diff
@@ -80,29 +80,29 @@ Adding plugins
...
 
-+---------------+--------------------------------------------------------------------+
++---------------+---------------------------------------------------------------------+
 |**Name**       |A short, speaking name that is displayed in the backend             |
-+---------------+--------------------------------------------------------------------+
-|**Key**        |A unique key to identify the module in your extension               |
-+---------------+--------------------------------------------------------------------+
++---------------+---------------------------------------------------------------------+
+|**Key**        |A unique key to identify the module in your extension                |
++---------------+---------------------------------------------------------------------+
```
Comment (liayn) [inline Documentation/Developer/Index.rst:99]: broken table

**Review by liayn — CHANGES_REQUESTED** [review]

**Review by liayn — APPROVED** [review]

**Pull request merged**